### PR TITLE
chore: update favicon and fix Netlify config

### PR DESCRIPTION
**Pull request by bdfinst**

### CODE REVIEW

File: website/static/favicons-new/android-chrome-512x512.png
Comment (btlghrants) [inline website/static/favicons-new/android-chrome-512x512.png:1]: Seems there's an _awful_ a lot whitespace in this image... izzat right?

File: website/static/favicons-new/android-chrome-maskable-512x512.png
Comment (btlghrants) [inline website/static/favicons-new/android-chrome-maskable-512x512.png:1]: Same as above -- lot's of whitespace..?

**Review by btlghrants — CHANGES_REQUESTED** [review]
I think the 512 images have some sizing issues..?  Maybe?  Glad to be wrong if that's not the case.

File: website/static/favicons-new/android-chrome-512x512.png
Comment (btlghrants) [reply website/static/favicons-new/android-chrome-512x512.png:1]: Figured -- and fixed -- reason.  The image generator in use (https://cthedot.de/icongen) needed the source SVG to have a width & height defined on it's <svg> tag.  For whatever reason.

File: website/static/favicons-new/android-chrome-maskable-512x512.png
Comment (btlghrants) [reply website/static/favicons-new/android-chrome-maskable-512x512.png:1]: Figured -- and fixed -- reason.  The image generator in use (https://cthedot.de/icongen) needed the source SVG to have a width & height defined on it's <svg> tag.  For whatever reason.

**Review by btlghrants — CHANGES_REQUESTED** [review]
LTGM, mostly.

What's the reason for the folder being `favicon-new` instead of just using the existing `favicon` folder though?  Happy to merge once I'm clear on that.

**Comment by bdfinst** [comment]
> LTGM, mostly.
> 
> What's the reason for the folder being `favicon-new` instead of just using the existing `favicon` folder though? Happy to merge once I'm clear on that.

I fixed that. Srry.

**Review by btlghrants — APPROVED** [review]
LGTM.

Gotta say too, that large-form dragon looks _frickin' sweet_ next to that old, fuzzy Dougie.  Big upgrade, yay.  :1st_place_medal:

**Pull request merged**